libs/common/database/src/lib/DataManagerBase.ts

```diff
@@ -2,6 +2,24 @@ import { deepFreeze } from '@genshin-optimizer/common/util'
 import type { Database } from './Database'
 import type { TriggerString } from './common'
 
+
```

Comment:
This is a base database class, that is being used for both Genshin Optimizer and Star Rail Optimizer, so it should not have genshin-specific artifact information. I am not 100% sure what the intention of the addition here is, but you'd would have to make it in the extension class ArtifactDataManager https://github.com/frzyc/genshin-optimizer/blob/master/libs/gi/db/src/Database/DataManagers/ArtifactDataManager.ts